sd_dynamic_prompts/dynamic_prompting.py

```diff
@@ -347,49 +355,31 @@ def ui(self, is_img2img):
         ]
 
     def process(
-        self,
-        p,
-        is_enabled: bool,
-        is_combinatorial: bool,
-        combinatorial_batches: int,
-        is_magic_prompt: bool,
-        is_feeling_lucky: bool,
-        is_attention_grabber: bool,
-        min_attention: float,
-        max_attention: float,
-        magic_prompt_length: int,
-        magic_temp_value: float,
-        use_fixed_seed: bool,
-        unlink_seed_from_prompt: bool,
-        disable_negative_prompt: bool,
-        enable_jinja_templates: bool,
-        no_image_generation: bool,
-        max_generations: int,
-        magic_model: str | None,
-        magic_blocklist_regex: str | None,
+            self,
+            p,
+            is_enabled: bool,
+            is_combinatorial: bool,
+            combinatorial_batches: int,
+            is_magic_prompt: bool,
+            is_feeling_lucky: bool,
+            is_attention_grabber: bool,
+            min_attention: float,
+            max_attention: float,
+            magic_prompt_length: int,
+            magic_temp_value: float,
+            use_fixed_seed: bool,
+            unlink_seed_from_prompt: bool,
+            disable_negative_prompt: bool,
+            enable_jinja_templates: bool,
+            no_image_generation: bool,
+            max_generations: int,
+            magic_model: str | None,
+            magic_blocklist_regex: str | None,
```

Comment:
Please revert the spurious formatting changes here :)

Reply:
Reverted.